diagnostic.md

```diff
@@ -36,18 +36,21 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+It's an relative path because its an instructed path on your local computer or
+working directory.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+You would want a relative path because if the font were taken down from the
+website for any reason, it could directly effect the outcome of your code.
```

Comment:
This is true and is a valid concern, but I think you went too far with your answer. If the font is hosted online we want an absolute path because it's not stored on our project so relative pathing wouldn't work.